cinema/views.py

```diff
@@ -127,20 +133,46 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                description="Filter by movie title",
+                type=str,
+            ),
+            OpenApiParameter(
+                name="genres",
+                description="Filter by genres "
+                            "(list of IDs, separated by comma)",
+                type={"type": "array", "items": {"type": "number"}},
+            ),
+            OpenApiParameter(
+                name="actors",
+                description="Filter by actors "
+                            "(list of IDs, separated by comma)",
+                type={"type": "array", "items": {"type": "number"}},
+            ),
+        ],
+        description="Retrieve a list of movies. "
+                    "Filter movies by title, genres and actors.",
+    )
+    def list(self, request, *args, **kwargs):
```

Comment:
The `list` method is correctly overridden to include the `@extend_schema` decorator, ensuring that the API documentation is generated for this endpoint.